semantic-conventions/src/tests/data/markdown/metrics_tables/expected.md

```diff
@@ -0,0 +1,76 @@
+
+## Common Attributes
+
+The following attributes SHOULD be included on all HTTP metrics for both server and client. 
+
+<!-- semconv metric.http -->
+| Attribute  | Type | Description  | Examples  | Required |
+|---|---|---|---|---|
+| `http.host` | string | The value of the [HTTP host header](https://tools.ietf.org/html/rfc7230#section-5.4). An empty Host header should also be reported, see note. [1] | `www.example.org` | See attribute alternatives |
+| `http.method` | string | HTTP request method. | `GET`; `POST`; `HEAD` | Yes |
+| `http.scheme` | string | The URI scheme identifying the used protocol. | `http`; `https` | See attribute alternative |
+| `http.status_code` | int | [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6). | `200` | No |
+
+**[1]:** When the header is present but empty the attribute SHOULD be set to the empty string. Note that this is a valid situation that is expected in certain cases, according the aforementioned [section of RFC 7230](https://tools.ietf.org/html/rfc7230#section-5.4). When the header is not set the attribute MUST NOT be set.
+<!-- endsemconv -->
+
+## HTTP Client 
+
+### HTTP Client Metrics
+
+<!-- semconv metric.http.client(metric_table,remove_constraints) -->
+| Name     | Instrument    | Unit (UCUM) | Description    |
+| -------- | ------------- | ----------- | -------------- |
+| `metric.http.client.duration` | Histogram | `ms` | Measures the duration of the outbound HTTP request. |
+<!-- endsemconv -->
+
+### HTTP Client Attributes
+
+The following attributes SHOULD be included on HTTP Client metrics, where applicable and available.
+
+<!-- semconv metric.http.client -->
+| Attribute  | Type | Description  | Examples  | Required |
+|---|---|---|---|---|
+| `net.peer.ip` | string | Remote address of the peer (dotted decimal for IPv4 or [RFC5952](https://tools.ietf.org/html/rfc5952) for IPv6) | `127.0.0.1` | See below |
+| `net.peer.name` | string | Remote hostname or similar, see note below. | `example.com` | See below |
+| `net.peer.port` | int | Remote port number. | `80`; `8080`; `443` | See below |
+
+**Additional attribute requirements:** At least one of the following sets of attributes is required:
+
+* `http.url`
+* `http.scheme`, `http.host`, `http.target`
+* `http.scheme`, `net.peer.name`, `net.peer.port`, `http.target`
+* `http.scheme`, `net.peer.ip`, `net.peer.port`, `http.target`
+<!-- endsemconv -->
+
+## HTTP Server
+
+### HTTP Server Metrics
+
+<!-- semconv metric.http.server(metric_table,remove_constraints) -->
+| Name     | Instrument    | Unit (UCUM) | Description    |
+| -------- | ------------- | ----------- | -------------- |
+| `metric.http.server.duration` | Histogram | `ms` | Measures the duration of the inbound HTTP request. |
+| `metric.http.server.active_requests` | UpDownCounter | `{requests}` | Measures the number of concurrent HTTP requests that are currently in-flight. |
```

Comment:
Could these be renamed by removing the `metric.` prefix?

Reply:
Yes, I agree, they could be. The `metric` prefix exists purely to stop clashes with the span semconv in code. So, we can strip it when rendering it to markdown.